Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+                            "votes": 0
+                        }, {
+                            "choice": "Objective-C",
+                            "url": "/questions/1/choices/3",
+                            "votes": 0
+                        }, {
+                            "choice": "Ruby",
+                            "url": "/questions/1/choices/4",
+                            "votes": 0
+                        }
+                    ]
                 }
+```
 
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
+The next resource is “Question”, it allows you to view a single question directly with an ID of a “Question”.
 
-    + Response 200
+```markdown
+## Question [/questions/{question_id}]
+```
 
-        [Gists Collection][]
+### URI Template
 
-    ### Create a Gist [POST]
-    To create a new Gist simply provide a JSON hash of the *description* and *content* attributes for the new Gist.
-
-    + Request (application/json)
-
-            {
-                "description": "Description of Gist",
-                "content": "String content"
-            }
-
-    + Response 201 (application/hal+json)
-
-        [Gist][]
-
-At this point you should be familiar with everything in this listing except the definition and discussion of the query parameter *"since"*:
-
-<a name="query_parameters"></a>
-#### Query Parameters
-
-    ## Gists Collection [/gists{?since}]
-
-As with the URI parameters the Query parameters are defined in the [URI Template][] format. The difference here is that a query parameter name is preceded by the *question mark* and it is always at the end of the URI Template.
-
-> **NOTE:** To define multiple query parameters simply comma-separate their names. For example `{?month,year}`.
-
-#### Action-specific Query Parameters
-
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
-
-Often a resource query parameter is specific to just one of the resource actions. In this case you can discuss it in the relevant action only. No matter where you discuss a query parameter it follows exactly the same syntax as with [URI Parameters][].
-
-## Subsequent Resource
-The last missing piece in the expected functionality is the Star Resource and its actions. Utilizing everything we have learned so far the Star Resource and its actions are defined as follows:
-
-    ## Star [/gists/{id}/star]
-    Star resource represents a Gist starred status.
-
-    The Star resource has the following attribute:
-
-    - starred
+The URI for the “Question” resource uses a variable component, expressed by [URI Template][]. In this case, we have an id variable called `{question_id}. Where `question_id` is substituted for a real `id`.
```

Comment:
Maybe something like:

> In this case, there is an ID variable called `question_id`, represented in the URI template as `{question_id}`. 

Do you need the last sentence? What is a "real ID?" 

Lastly, might want to sync on lowercase vs. uppercase `ID`.